model/task_lifecycle_test.go

```diff
@@ -1190,6 +1191,66 @@ func TestAbortTask(t *testing.T) {
 	})
 
 }
+func TestTryDequeueAndAbortCommitQueueVersionWithBlacklist(t *testing.T) {
```

Comment:
do we need a new, separate, test? Could the check for -1 priority be folded into the next test?

Reply:
I can't do that unless I change the other test to using CLI Patches.